numbers/index.js

```diff
@@ -81,10 +81,11 @@ export const roundDown = (num) => {
   converted to a number
 */
 export const convertToInt = (str) => {
-  if (Number.isNaN(Number(str)) || typeof str !== 'string') {
+  var parInt = Number.parseInt(str, 10);
+  if (Number.isNaN(parInt) || typeof str !== 'string') {
     return false;
   }
-  return Math.round(Number(str)); // this took me by far the longest
+  return Number(parInt);
```

Comment:
As we've previously done an `isNaN` check, we can actually just return the result of parInt because we are now certain it's a number - no need to cast if we defo know it's a number :)

Reply:
very good point!